test/models/item_test.rb

```diff
@@ -3,4 +3,29 @@
 class ItemTest < ActiveSupport::TestCase
   should belong_to :itemable
   should belong_to :skill_set
+
+  test "category and cat_attributes test" do
+    blacksmith = Category.create(name: "blacksmith")
+    armory = Category.create(name: "armory")
+    weapon_ss = SkillSet.create(strength: 1)
+    armor_ss = SkillSet.create(speed: -1)
+
+    i1 = Item.create(name: "sword1", category: blacksmith, skill_set: weapon_ss)
+    i2 = Item.create(name: "sword2", category: blacksmith, skill_set: weapon_ss )
```

Comment:
Useless assignment to variable - `i2`.<br>Line is too long. [81/80]<br>Space inside parentheses detected.